### PR TITLE
Scoped view highlight & OTel support

### PR DESCRIPTION
**Pull request by meda123**
A few changes on this PR: Scoped view should NOT be at the bottom. We want to call out the option that doesn't require setup right away. The image in the scoped view should be changed because it doesn't actually show scoped view (will submit image on Slack thread). We're going to be providing OTel support (data type) so I updated the main sentence under the scoped views section.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/errors-inbox/errors-inbox.mdx [hunk 1]
```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox directly from scoped views [#scoped]
```
Comment (obarney) [inline src/content/docs/errors-inbox/errors-inbox.mdx:54]: A couple things:

1.   I have noticed that capitalization is inconsistent across this entire page.   It seems to be either `errors inbox` or `Errors inbox`.  I think it should be the latter or both words should be capitalized. 
1.   I would say "... from entity scoped..."

File: src/content/docs/errors-inbox/errors-inbox.mdx [hunk 2]
```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
...
+
+Scoped views allow you to view errors for entities in browser, mobile, and [Open Telemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/#benefits). Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **Errors inbox** in the left nav.
```
Comment (obarney) [inline src/content/docs/errors-inbox/errors-inbox.mdx:56]: Perhaps start with:
"[Entity](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) scoped views allow you to view the errors for a single APM, Browser, Mobile, or [Open Telemetry]() application."

Consider adding "... in the left nav under **Triage**"

**Comment by bradleycamacho** [comment]
@obarney Addressed all those in the latest commit!

File: src/content/docs/errors-inbox/errors-inbox.mdx [hunk 3]
```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
...
+  title="An image showing how to navigate to a scoped browser entity"
+  alt="This is an image of the main errors inbox UI inside a browser entity"
+  src={scopedBrowser}
+/>
+
+<figcaption>Traige errors for a browser or mobile entity by selecting **Errors inbox** in the left nav of the browser or mobile explorer.</figcaption>
+
+NOTE: Please replace image above with new browser image provided on Slack channel 
+
+
```
Comment (obarney) [inline src/content/docs/errors-inbox/errors-inbox.mdx:68]: It feels a bit redundant to me to have basically the same image twice.  Any reason not to add Otel (and APM??) to the list <figcaption> above?

File: src/content/docs/errors-inbox/errors-inbox.mdx [hunk 4]
```diff
@@ -49,11 +51,37 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox from entity scoped views [#scoped]
+
+Entity scoped views allow you to view errors for a single APM, browser, mobile, or [Open Telemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/#benefits) application. Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **errors inbox** in the left nav under **triage**.
```
Comment (obarney) [inline src/content/docs/errors-inbox/errors-inbox.mdx:56]: I still like the the idea of having the word Entity link to: https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/
but if we think that is a distraction from the main point I am ok leaving it out.

**Review by obarney — COMMENTED** [review]
In the menu in NR1 we call it `Errors inbox` so I think it's strange to have it all lower case here.     @meda123 or Adam should weigh in on this.

I also added a comment about the images seeming redundant.

File: src/content/docs/errors-inbox/errors-inbox.mdx (hunk 3 above)
Comment (bradleycamacho) [reply src/content/docs/errors-inbox/errors-inbox.mdx:68]: I agree. I'd rather use Otel here and then just list APM/Browser in  relevant section above:


Paragraph:
`Access the errors inbox view for a specific entity by navigating to that entity in the APM, browser, mobile, or OpenTelemetry monitoring view and selecting **errors inbox** in the left nav under **triage**`

Caption:
`Traige errors for an entity by selecting **errors inbox** in the left nav of the explorer.`

Comment (meda123) [reply src/content/docs/errors-inbox/errors-inbox.mdx:68]: Good point, sound good to me! Thanks Oscar.

File: src/content/docs/errors-inbox/errors-inbox.mdx (hunk 4 above)
Comment (bradleycamacho) [reply src/content/docs/errors-inbox/errors-inbox.mdx:56]: We try not to link out to "basic" NR concepts such as entity, but I believe that it does make sense in this specific situation. I'll add a link.

Comment (meda123) [reply src/content/docs/errors-inbox/errors-inbox.mdx:56]: +1 for that. We often assume our users known these definitions but it doesn't hurt to clarify.

**Comment by bradleycamacho** [comment]
> In the menu in NR1 we call it `Errors inbox` so I think it's strange to have it all lower case here. @meda123 or Adam should weigh in on this.

We have a capitalization guide for all NR language: https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage#what-not-to-capitalize

Errors inbox is sentence cased in the UI and in the doc when it's a direct reference to the UI. Otherwise, it is always under-case. I just committed a few fixes.

File: src/content/docs/errors-inbox/errors-inbox.mdx (hunk 4 above)
Comment (meda123) [reply src/content/docs/errors-inbox/errors-inbox.mdx]: @obarney - the style docs ask that we refer to the product as errors inbox when used in the middle of a sentence https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/

**Comment by obarney** [comment]
Ok, I understand now.  Thank you @bradleycamacho

**Pull request merged**